demo/context-menu-items-demos.html

```diff
@@ -20,7 +20,7 @@ <h3>Nested Menu Items</h3>
             contextMenu.items = [
               {text: 'Menu Item 1', children:
                 [
-                  {text: 'Menu Item 1-1', checkable: true, checked: true},
+                  {text: 'Menu Item 1-1 very long item', checkable: true, checked: true},
```

Comment:
One case related to the long item is still not fixed:

![long-item](https://user-images.githubusercontent.com/10589913/70629287-68c67300-1c32-11ea-982f-495b440e130a.png)

Reply:
because there is a limited vertical line, and nested sub-menu should not be outside the line.
I did set the nested sub-menu is Math.max(the coordinate of the line, and the x of nested submenu overlay ) by: 
`style.left = Math.max(overlay.getBoundingClientRect().x,
                parent.$.overlay.getBoundingClientRect().x - overlay.getBoundingClientRect().width) + 'px';`
If it need to set Math.max (0,...), it will be outside, and goes to left side, and will lost the content of nested sub-menu. It just is my idea.